DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -36,5 +37,86 @@ public void Get_course_content_should_return_course()
             };
             result.Should().BeEquivalentTo(expectedCourse);
         }
+
+        [Test]
+        public void Get_progress_id_should_return_progress_id()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            var result = courseContentService.GetProgressId(candidateId, customisationId);
+
+            // Then
+            var expectedProgressId = 10;
+            result.Should().Be(expectedProgressId);
+        }
+
+        [Test]
+        public void Update_login_count_should_not_increment_login_if_no_new_session()
+        {
+            // When
+            const int candidateId = 9;
+            const int customisationId = 259;
+            var progressId = courseContentService.GetProgressId(candidateId, customisationId);
+            var expectedLoginCount = CourseContentTestHelper.GetLoginCount(progressId);

```

Comment:
The lines above this comment should be grouped under `// Given`. The tests so be split into `// Given` (setup), `// When` (execute the method you want to test) and `// Then` (check the result) 🙂